pkgs/servers/knxd/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, buildPackages
+, autoreconfHook, pkg-config, indent, perl
+, argp-standalone, fmt, libev
+, withSystemd ? stdenv.isLinux, systemd
+, withUsb ? stdenv.isLinux, libusb1
```

Comment:
Why do we need the libusb option?

Reply:
It fails on darwin with libusb enabled:
```sh
knxd> checking for libusb-1.0 >= 1.0.9 ... yes
knxd> configure: error: not all needed functions for USB backend available
```

Reply:
@SuperSandro2000 is it OK?

Reply:
not sure if we should just make depended on linux

Reply:
Well, should we drop darwin support and keep `libusb1` always enabled?